### ISSUE TITLE
Ability to open on new browser window

### ISSUE THREAD

**Issue by pablocobelli** [suan/instant-markdown-d#34]
First of all, let me say that this is a very useful piece of software, thanks for that. 
I use it along with your vim-instant-markdown. I've seen that it will open the preview in a new _tab_ (if there's a browser already open), and that could sometimes be impractical, particularly when that browser is open in another desktop (I'm using OS X). 

I've seen that this can be solved by changing line 97 on instant-markdown-d to:

```
exec('osascript -e \'tell application "Safari" to make new document with properties    {URL:"http://localhost:8090"}\'', function(error, stdout, stderr){});
```

This will open the preview on a new window instance at the current desktop. I think this can also be extended to open the _user's default browser_ instead of safari (as in my code above). 

Please tell me if you think this is worth it (useful for the rest of the users) and I'll prepare a PR.

**Comment by Migacz85** [comment]
I want to use it :-) I'm experiencing exactly the same thing happening it would be nice to have it configurable form vim. - something like opening_in_new_browser = true and that's it.

**Comment by ashwinvis** [comment]
Fixed in #60, you can configure custom command like `instant-markdown-d --browser "firefox --new-window"`
